app/api/helpers/calendar/ical.py

```diff
@@ -40,8 +40,8 @@ def to_ical(event, include_sessions=False, my_schedule=False, user_id=current_us
         if my_schedule:
             if not (current_user or user_id):
                 return jsonify(error='Forbidden Access'), 401
-            if not user_id:
-                user_id = current_user.id
+
+            user_id = current_user.id if not user_id else user_id
```

Comment:
user_id overrides current_user

Reply:
Sir I have set it to current_user.id if user_id is not set :sweat_smile: so user_id will override current_user